### DISCUSSION TITLE
The Differences between SaaS Build Kit(sbt) and SaaS Boost？

### DISCUSSION THREAD

**Discussion by houweilong** [awslabs/sbt-aws#103] [Q&A] [answered]
Hi， I am a developer for saas application, I wan to know the Differences between SaaS Build Kit(sbt) and SaaS Boost？I read their documentation and was very confused.. Thanks.

**Comment by tobuck-aws** [comment]
SaaS Boost was a product primarily targeted at migration scenarios, where customers wanted to quickly containerize their existing solutions and transition to operating in a SaaS business model. It provided a turnkey solution to help customers migrate their applications to a containerized environment and adopt SaaS operational practices.

In contrast, SBT is a more technical offering with a broader range of use cases, including greenfield SaaS development. Unlike SaaS Boost's turnkey approach, SBT requires development effort from customers to compose and integrate the provided building blocks into their SaaS solutions.

It's important to note that while SaaS Boost is no longer supported, the knowledge and best practices gained from its development have contributed to the evolution of SBT.

**Reply by houweilong** [reply, marked as answer]
Okay, I understand. Thank you for your reply.
